### PR TITLE
DATAUP-330: Fixing empty specs in the dynamicTable and loadingWidget tests

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

Adding expectations to tests which were not testing things adequately, and disabling some empty tests.

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-X
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [x] Changes available by running `make test-frontend-unit`

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook

### CODE REVIEW

File: test/unit/spec/loadingWidgetSpec.js [hunk 1]
```diff
@@ -1,52 +1,66 @@
...
+    $,
+    LoadingWidget,
+    LoadingTemplate
+) => {
     'use strict';
 
+    const templateHtml = LoadingTemplate.replace(
+        '{{ static_url("kbase/images/kbase_animated_logo.gif") }}',
+        '/narrative/static/kbase/images/kbase_animated_logo.gif'
+    );
```
Comment (ialarmedalien) [inline test/unit/spec/loadingWidgetSpec.js:11]: pull in the template for the loading section -- otherwise any DOM checks will be pointless as the widget relies on a certain DOM structure being in place.

File: test/unit/karma.conf.js [hunk 2]
```diff
@@ -31,41 +31,44 @@ module.exports = function (config) {
             'nbextensions/appcell2/widgets/tabs/*.js': ['coverage'],
             'kbase-extension/static/kbase/js/*.js': ['coverage'],
         },
+        // karma defaults:
+        // included: true; nocache: false; served: true; watched: true;
         files: [
-            'kbase-extension/static/narrative_paths.js',
+            'node_modules/jasmine-ajax/lib/mock-ajax.js',
```
Comment (ialarmedalien) [inline test/unit/karma.conf.js:37]: reordered these lines and added a line to serve templates:

```js
kbase-extension/kbase_templates/*.html
```

File: test/unit/spec/dynamicTableSpec.js [hunk 3]
```diff
@@ -50,24 +50,35 @@ define(['jquery', 'bluebird', 'widgets/dynamicTable'], ($, Promise, DynamicTable
...
+        xit('Should decorate rows properly', () => {});
 
-        it('Should add outer widget classes properly', () => {});
+        xit('Should add outer widget classes properly', () => {});
 
-        it('Should add table styles properly', () => {});
+        xit('Should add table styles properly', () => {});
 
-        it('Should have optional download buttons', () => {});
+        xit('Should have optional download buttons', () => {});
```
Comment (briehl) [inline test/unit/spec/dynamicTableSpec.js:82]: :+1:

**Review by briehl — APPROVED** [review]
Looks good.
